grails-app/controllers/com/unifina/controller/api/CommunityProductApiController.groovy

```diff
@@ -100,6 +102,72 @@ class CommunityProductApiController {
 		render(status: 204)
 	}
 
+	@StreamrApi
+	def findCommunitySecrets(String communityAddress) {
```

Comment:
Convention: let's create `CommunitySecretController`

- This also allows you to use simpler action names like `find` instead of `findCommunitySecret`